src/Umbraco.Infrastructure/Persistence/Dtos/ContentVersionDto.cs

```diff
@@ -20,6 +20,7 @@ public class ContentVersionDto
     [Column("nodeId")]
     [ForeignKey(typeof(ContentDto))]
     [Index(IndexTypes.NonClustered, Name = "IX_" + TableName + "_NodeId", ForColumns = "nodeId,current", IncludeColumns = "id,versionDate,text,userId")]
+    [Index(IndexTypes.NonClustered, Name = "IX_" + TableName + "_NodeIdV2", ForColumns = "nodeId,current", IncludeColumns = "versionDate,preventCleanup,text,userId")]
     public int NodeId { get; set; }
```

Comment:
This one just needs the prevent cleanup column added. No need to have both indexes

Reply:
@nzdev , then which name should be assigned to this index: IX_umbracoContentVersion_NodeIdV2 or just IX_umbracoContentVersion_NodeId?

Reply:
IX_umbracoContentVersion_NodeId.  Could delete the existing index and recreate it.